### ISSUE TITLE
[1.3.0-RC3] Breakage in ";" multi-command support

### ISSUE THREAD

**Issue by dwijnand** [sbt/sbt#4897]
## steps
sbt version: 1.3.0-RC3

See https://github.com/playframework/play-samples/pull/43 which uses

```
sbt ++$TRAVIS_SCALA_VERSION ";project gatling;gatling:compile"
```

and

```
sbt ++$TRAVIS_SCALA_VERSION ";project gatling; gatling:compile"
```

## problem

```
[error] Expected ';'
[error] No such setting/task
[error] gatling:compile
[error]                ^
```

## expectation

No regression in behaviour.

**Comment by eed3si9n** [comment]
@dwijnand Thanks for turning it into issue.

**Comment by eed3si9n** [comment]
@eatkins fixed this already in #4871; the following nightly works.

```scala
sbt.version=1.3.0-bin-20190723T192002
```
